authentikos/go.mod

```diff
@@ -3,74 +3,76 @@ module istio.io/test-infra/authentikos
 go 1.18
 
 require (
-	github.com/Masterminds/sprig/v3 v3.2.2
+	github.com/Masterminds/sprig/v3 v3.2.3
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/oauth2 v0.0.0-20221014153046-6fdb5e3db783
-	google.golang.org/api v0.107.0
-	k8s.io/api v0.24.3
-	k8s.io/apimachinery v0.24.3
-	k8s.io/client-go v0.24.3
+	golang.org/x/oauth2 v0.7.0
+	google.golang.org/api v0.114.0
+	k8s.io/api v0.24.2
+	k8s.io/apimachinery v0.24.2
+	k8s.io/client-go v0.24.2
```

Comment:
These look to be downgraded.

Reply:
And sorry I missed this before approving the other PR (which I added a DNM to but not soon enough I guess).

Reply:
Sorry forgot to include the context - its downgraded to meet the version of the other modules. I was thinking this would improve caching. See https://blog.howardjohn.info/posts/go-build-times/, I have become a bit obsessed with build caching...